Packs/CortexXDR/Scripts/XDRConnectedEndpoints/XDRConnectedEndpoints.py

```diff
@@ -0,0 +1,39 @@
+import traceback
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    try:
+        args = demisto.args()
+        last_seen_gte = args.get('from')
+        last_seen_lte = args.get('to')
+        limit = args.get('limit', '100')
+
+        get_endpoints_args = {'limit': limit}
+
+        get_endpoints_args = {}
+        if last_seen_gte:
+            get_endpoints_args['last_seen_gte'] = last_seen_gte
+        if last_seen_lte and last_seen_lte != '0001-01-01T00:00:00Z':
```

Comment:
What is the meaning of `0001-01-01T00:00:00Z` here? where does it come from?

Reply:
Then I would suggest adding a dev comment to state that's the default value it receives.